4_cv_basics/2_playing_with_images/Makefile

```diff
@@ -5,9 +5,9 @@ CC = g++
 PROJECT = playing_with_images		
 
 # Here Include specifies which directories can be added in the include path of the main.cpp
-CFLAGS = -std=c++11 $(shell pkg-config --cflags opencv4) -Iinclude/		
+CFLAGS = -std=c++11 $(shell pkg-config --cflags opencv4) $(shell pkg-config --cflags sdl2)		
```

Comment:
Same here, were cflags necessary here?

Reply:
Here the cflags is not necessary since we are not using the sdl2 library in this folder's code.

Reply:
> Here, the cflags is not necessary since we are not using the sdl2 library in this folder's code.

@Ritika128 make the necessary changes in the next commit